clustertask/validate-environment/templates/clustertask.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: tekton.dev/v1beta1
+kind: ClusterTask
+metadata:
+  name: {{ include "stakater-validate-environment.name" . }}
+  annotations:
+    description: |
+      Validate Helm Release Status of Tronodor Environment
```

Comment:
@mustafaStakater should we use the word HelmRelease here? Wouldn't it be better we just use word Environment